public/react-services/generic-request.js

```diff
@@ -28,18 +28,23 @@ export class GenericRequest {
       const { timeout } = wazuhConfig.getConfig();
       const requestHeaders = {
         'Content-Type': 'application/json',
-        'kbn-xsrf': 'kibana'
+        'kbn-xsrf': 'kibana',
       };
       const tmpUrl = getHttp().basePath.prepend(path);
 
-      try{
-        requestHeaders.pattern = (await getDataPlugin().indexPatterns.get(AppState.getCurrentPattern())).title;
-      }catch(error){};
+      try {
+        requestHeaders.pattern = (
+          await getDataPlugin().indexPatterns.get(AppState.getCurrentPattern())
+        ).title;
+      } catch (error) {
+        throw new Error('Error obtaining the current index patten', error);
+      }
 
       try {
         requestHeaders.id = JSON.parse(AppState.getCurrentAPI()).id;
       } catch (error) {
         // Intended
+        throw new Error('Error obtaining the current API', error);
```

Comment:
**thought**: There is a comment with the `Intended`, I am not sure if it throws an error here, could break something.

Anyways the second argument is not used by the `Error` class constructor.